ajax/libs/pickmeup/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "pickmeup",
+  "filename": "pickmeup.min.js",
+  "description": "Really simple, powerful, customizable and lightweight standalone datepicker",
+  "keywords": [
+    "datepicker",
+    "date",
+    "calendar"
+  ],
+  "version": "3.0.11",
+  "homepage": "https://cleverstyle.org/",
+  "author": {
+    "name": "Nazar Mokrynskyi",
+    "email": "nazar@mokrynskyi.com",
+    "url": "https://cleverstyle.org"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nazar-pc/PickMeUp.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nazar-pc/PickMeUp.git",
+    "fileMap": [
+      {
+        "basePath": "css",
+        "files": [
+          "pickmeup*.+(css)"
+        ]
+      },
+      {
+        "basePath": "js",
```

Comment:
We don't use `js`as the bathPath.
Thanks

Reply:
I think that those files are hosted by js pattern, do not them?
jquery.pickmeup.twitter-bootstrap.js and jquery.pickmeup.uikit.js